arches/app/search/elasticsearch_dsl_builder.py

```diff
@@ -98,7 +98,7 @@ def search(self, index="", **kwargs):
             return self.se.search(index=index, scroll=self.scroll, **self.dsl)
 
     def count(self, index="", **kwargs):
-        return self.se.count(index=index, body=self.dsl)
+        return self.se.count(index=index, **self.dsl)
 
     def delete(self, index="", **kwargs):
         return self.se.delete(index=index, body=self.dsl, **kwargs)

```

Comment:
Huh. I guess I left this one out because it wasn't mentioned in the github issue I linked. As we discussed, that's not the most authoritative source, so I went to the elastic docs.

Here's the [migration guide](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/migration.html#migration-keyword-only-args) for 8. Curiously, they're saying that the deprecation period was from v7.14 and this shouldn't have been working as of v8. Are these perhaps really obscure code paths that people just aren't using?

I'll update the delete methods, given the guidance in the migration guide 👍

Reply:
Ah, right, of course it's still working, otherwise I wouldn't have even seen the deprecation warnings.